generate/beat/{beat}/NOTICE

```diff
@@ -0,0 +1,5 @@
+{beat}
+Copyright 2017 {full_name}
```

Comment:
We should introduce {year} here so we have it always up-to-date ;-) Not needed in this PR.